apps/src/templates/teacherNavigation/EmptyState.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import {Heading3, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+
+import styles from './teacher-navigation.module.scss';
+import dashboardStyles from '@cdo/apps/templates/teacherDashboard/teacher-dashboard.module.scss';
+
+export interface EmptyStateContent {
+  headline: string;
+  descriptionText: string | null;
+  imageComponent: JSX.Element;
+  button: JSX.Element | null;
+}
+
+interface EmptyStateProps {
+  emptyStateDetails: EmptyStateContent;
+}
+
+export const EmptyState: React.FC<EmptyStateProps> = props => {
+  const {emptyStateDetails} = props;
+
+  const imageComponent = emptyStateDetails?.imageComponent;
+  const headline = emptyStateDetails?.headline;
+  const descriptionText = emptyStateDetails?.descriptionText;
+  const button = emptyStateDetails?.button;
```

Comment:
```suggestion
export interface EmptyStateProps {
  headline: string;
  descriptionText: string | null;
  imageComponent: JSX.Element;
  button: JSX.Element | null;
}

export const EmptyState: React.FC<EmptyStateProps> = ({ imageComponent, headline, descriptionText, button}) =>
```

What about something like this? And then you can create empty state with `<EmptyState {...emptyStateContent} />`